play-json/shared/src/main/scala/play/api/libs/json/Reads.scala

```diff
@@ -366,80 +366,6 @@ trait DefaultReads extends LowPriorityDefaultReads {
     }
   }
 
-  /**
```

Comment:
So the JS is no longer covered ?

Reply:
JS BigDecimal/BigInt readers are basically the same.

Reply:
The code was moved to js specific directory from here.

Reply:
Anyway, it was cover by shared, and is no longer (not good)